docs/content/contributing.md

```diff
@@ -0,0 +1,6 @@
+title = "Contributing to Bartholomew"
+date = "2022-05-08T14:05:02.118466Z"
+
+[extra]
+url = "https://github.com/fermyon/bartholomew/blob/main/docs/content/contributing.md"
```

Comment:
We may want to file an issue/reminder to populate this page.

Reply:
I _think_ we could construct this using `https://github.com/fermyon/bartholomew/blob/main/docs/content` as base?